lab/README.md

```diff
@@ -16,30 +16,29 @@ The steps below will get you up and running to deploy the Relecloud Django appli
 ## Get Started
 ### Create your own repo from this template
 
-Open the [Azure-Samples/azure-django-postgres-aca](https://github.com/Azure-Samples/azure-django-postgres-aca) repository and click the "Use this Template" button at the top right hand corner.
+Open the [Azure-Samples/azure-django-postgres-aca](https://github.com/Azure-Samples/azure-django-postgres-aca) repository, click the **Use this Template** button at the top right hand corner, and select **Create a new repository**.
 
 ![The green "Use this Template" button in the top right section of the repo page on GitHub](../assets/use-this-template.png)
 
-Once you have created the repo, launch the repo in GitHub Codespaces or clone it to your local machine.
+Once you have created the repo, launch the repo in GitHub Codespaces or clone it to your local machine:
 
 ### [Option 1]: Launch in GitHub Codespaces
 
 You can use [GitHub Codespaces](http://aka.ms/codespaces) to launch your repo in a browser-based VS Code environment. This is a great way to get started with no local setup required. GitHub users recieve 60 free hours of Codespaces per month.
 
-Select the green "Code" button in the top right section of the repo page on GitHub and select "Open with Codespaces". If you haven't created a Codespace before, you will be prompted to create one. Once created, you will be taken to your Codespace environment.
+Select the green **Code** button in the top right section of the repo page on GitHub and select **Open with Codespaces**. If you haven't created a Codespace before, you will be prompted to create one. Once created, you will be taken to your Codespace environment.
 
 ![Create a Codespace](../assets/create%20a%20codespace.png)
 
-You can choose to work from inside your browser or open that Codespace using your local Visual Studio Code.
+You can choose to work inside your browser or open that Codespace using your local Visual Studio Code.
 
-### [Option 2]: Clone to Your Local Machine and Open as a Dev Container
+### [Option 2]: Clone to your local machine and open as a Dev Container
 
-Open **your repo** that you have created from Azure-Samples/azure-django-postgres-aca, click Code at the top right, and on the Local tab and copy the clone command for `HTTPS`, `SSH`, or `GitHub CLI` as appropriate.
+1. Open **your repo** that you have created from Azure-Samples/azure-django-postgres-aca, click Code at the top right, and on the Local tab, copy the clone command for `HTTPS`, `SSH`, or `GitHub CLI` as appropriate.
+2. Run the clone command on your local machine to clone the repo locally. 
+3. Open the repo inside VS Code with the [Dev Containers extension](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-containers). ⚠️ Unfortunately, If you're on an Apple M1/M2, it's not currently possible to open this project in a Dev Container. Use GitHub Codespaces or a virtual environment instead.
```

Comment:
M1 note

Reply:
Yeah we were working to see if we could resolve the M1 issue. I am not sure that virtual environment would work since the postgreSQL db. I would perhaps just suggest to use Codespaces instead.

```suggestion
3. Open the repo inside VS Code with the [Dev Containers extension](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-containers). ⚠️ Unfortunately, If you're on an Apple M1/M2, it's not currently possible to open this project in a Dev Container. Please use GitHub Codespaces to ensure that you have all the necessary dependencies and configurations.
```

Reply:
True yeah it only worked for me as I also have Postgres local running. Will use your wording.